test/e2e/multi-node-3.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -e
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $BASEDIR
```

Comment:
```suggestion
cd "$BASEDIR"
```
Without quotes this will fail if script is located in a directory with a space. In general I prefer always quoting variables